### PR TITLE
WIP- Initial Project setup | configs files | cosmetic changes 

### PR DESCRIPTION
**Pull request by bhumitaylor14**
## Description
WIP commit  - UI integration initial setup rollup config , editor config, suggested changes 

## Motivation and Context
Develop Etherspot UI library for sending native token.

## How Has This Been Tested?
On local machine

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ tick] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # Changelog
 
+## [0.1.0] - 2023-12-05
+
+### Added
+- added package.json and updated dependancies in it.
+- added contributing , changelog, jest config, security, rollup config, ts config
```
Comment (IAmKio) [inline CHANGELOG.md]:
```suggestion
- added contributing, changelog, jest config, security, rollup config, ts config
```

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+dist/
```
Comment (IAmKio) [inline .gitignore:3]: Please find a better .gitignore for node.js / javascript projects

File: package.json [hunk 3]
```diff
@@ -1,29 +1,85 @@
...
+    "moment": "^2.29.4",
+    "react-content-loader": "^6.2.1",
+    "react-icons": "^4.4.0",
+    "react-ios-switch": "^0.1.19",
+    "react-router-dom": "^6.15.0",
+    "react-spinners": "^0.13.2",
+    "reflect-metadata": "^0.1.13",
+    "rollup-plugin-postcss": "^4.0.2",
+    "rxjs": "^6.6.7",
+    "styled-components": "^5.3.5"
```
Comment (IAmKio) [inline package.json]: Please version lock these dependencies. Remember when you're building a library, we need to provide stability as a guarantee to the other developers

File: tsconfig.json
Comment (IAmKio) [inline tsconfig.json:1]: Please remove the dead code / config here

**Review by IAmKio — CHANGES_REQUESTED** [review]

**Comment by bhumitaylor14** [comment]
@kio suggested changes done

**Review by IAmKio — APPROVED** [review]

**Pull request merged**